backend/tests/unit/test_all_users_query.py

```diff
@@ -0,0 +1,35 @@
+from app.graphql.all_users import AllUsersQuery
+import graphene
+from graphene.test import Client
+from app.resources.user_dto import UserDTO
+
+
+def test_all_users(mocker):
+    mock_result = [UserDTO('1', 'John', 'Doe', 'JohnDoe@email.com', 'Admin'), UserDTO('2', 'Jane', 'Doe', 'JaneDoe@email.com', 'ASP')]
+
+    mocker.patch(
+        "app.services.implementations.user_service.UserService.get_users",
+        return_value=mock_result,
+    )
+
+    schema = graphene.Schema(query=AllUsersQuery)
+    client = Client(schema)
+    executed = client.execute(
```

Comment:
This looks like a GraphQL test, maybe we should move it to under `tests/graphql` instead